sklearn/model_selection/tests/test_validation.py

```diff
@@ -1647,3 +1652,109 @@ def test_sample_weight():
     # counts.
     assert train_metric == exp_train_pr
     assert test_metric == exp_test_acc
+
+
+@pytest.mark.parametrize("metric,greater_is_better,needs_proba,sample_wt", [
+    (accuracy_score, True, False, [1, 999999, 1, 999999]),
+    (accuracy_score, True, False, [100000, 200000, 100000, 200000]),
+    (accuracy_score, True, False, [100000, 100000, 100000, 100000]),
+    (accuracy_score, True, False, [200000, 100000, 200000, 100000]),
+    (accuracy_score, True, False, [999999, 1, 999999, 1]),
+    (accuracy_score, True, False, [2000000, 1000000, 1, 999999]),
+    (precision_score, True, False, [2000000, 1000000, 1, 999999]),
+    (log_loss, False, True, [2500000, 500000, 200000, 100000]),
+    (brier_score_loss, False, True, [2500000, 500000, 200000, 100000]),
+])
```

Comment:
Should we test more scorers?

Reply:
I don't think it's necessary.  We could, but this is way better than what seems to currently exist in sklearn.  I also have https://gist.github.com/deaktator/94545f807f139eba2c8a15381f2495e0 for which I'd like to submit a future PR to sklearn.  It's just test code.  Not any change to implementation.